mobile/src/main/java/org/hawkular/client/android/Animae/MyBounceInterpolator.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
1. Never use camel case for package name
1. Let's rename the package for _animation_